crates/burn-jit/src/fusion/on_write/trace.rs

```diff
@@ -383,6 +421,18 @@ impl FuseOnWriteTrace {
             SequenceArg::new(),
             SequenceArg::new(),
             SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
+            SequenceArg::new(),
```

Comment:
Not sure if there is a better way to do this 😅

Reply:
It's simply that each precision has its state. It would be cleaner to have a map `Map<ElemenType, State>`, instead of listing all posibilities. I'll think how I can refactor it, but not in this PR.